src/components/Tweets/Tweet.js

```diff
@@ -0,0 +1,20 @@
+import React, { PropTypes } from 'react';
+import cx from 'classnames';
+
+import styles from './Tweet.scss';
+
+const Tweets = (props) => {
+  return (
+      <div className={cx(styles.block)}>
+        <div className={cx(styles.image)}>
+          <img width="48" height="48" src={props.user.profile_background_image_url}/>
```

Comment:
Can you add a config.js file in /src/ which exports just a simple object? Then you can move avatarSize as a parameter in there, so this value is configurable.